src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_help.py

```diff
@@ -321,6 +321,22 @@
         - name: Cancel an operation.
           text: az sql elastic-pool op cancel -g mygroup -s myserver -ep myelasticpool -n d2896db1-2ba8-4c84-bac1-387c430cce40
     """
+helps['sql failover-group'] = """
```

Comment:
What would be the command group name for managed instance failover group? Maybe `mi-failover-group`?

Reply:
I just want to have a plan ready for the future so that we don't realise later that we messed up the design of these commands :)

Reply:
I don't know, but your proposition sounds reasonable